.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+repos:
+- repo: https://github.com/compilerla/conventional-pre-commit
+  rev: v1.3.0
+  hooks:
+    - id: conventional-pre-commit
+#      stages: [commit-msg]
+      args: [] # optional: list of Conventional Commits types to allow
+- repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
We should add a doc somewhere to give instructions about how configure local hooks.